src/box/space.h

```diff
@@ -250,6 +251,11 @@ struct space {
 	struct rlist memtx_stories;
 	/** Space upgrade state or NULL. */
 	struct space_upgrade *upgrade;
+	/**
+	 * Pointer to WAL extensions (i.e. if not NULL WAL entries
```

Comment:
Not "to WAL extensions", but to the WAL extension *configuration* for this space. I think that we should rename `struct wal_ext` to `space_wal_ext`.

Reply:
There's `space_upgrade` in `struct space`, which has the prefix. The reason why I recommend calling it `space_wal_ext` is that in future we might want to introduce `struct wal_ext` for storing the global WAL extension configuration. This would result in a name conflict with the existing `struct wal_ext`. In fact, we could already store the space -> cfg hash table in `struct wal_ext` in the EE part - it could simplify the interaction between Lua and C. `index` and `constraint` don't need this prefix, because there can't be index or constraint without a space. With `wal_ext` it isn't so - there's an entity called WAL and `wal_ext` sounds like it belongs to it while in your patch it stores a space configuration. The methods could be named `space_wal_ext_by_name` and `space_wal_ext_process_request`.